q2_mlab/learningtask.py

```diff
@@ -0,0 +1,268 @@
+import json
+import numpy as np
+import pandas as pd
+import time
+
+# CV Methods
+from sklearn.model_selection import RepeatedStratifiedKFold
```

Comment:
No, that is something we should still use (ideally) for regression. It looks like calour is available in conda (bioconda), so it can be specified as a dependency and we can import that KFold class.

Reply:
That sounds good! Make sure to also list in setup.py.